workflows/eager.nf

```diff
@@ -270,6 +280,21 @@ workflow EAGER {
         ch_dedupped_flagstat = Channel.empty()
     }
 
+    //
+    // MODULE QUALIMAP
+    //
+
+    if ( !params.skip_qualimap ) {
+       ch_qualimap_input = ch_dedupped_bams
+            .map {
+            meta, bam, bai ->
+                [ meta, bam ]
+            }
+        QUALIMAP_BAMQC(ch_qualimap_input, ch_snpcapture_bed)
+        ch_versions      = ch_versions.mix( QUALIMAP_BAMQC.out.versions )
+        ch_multiqc_files = ch_multiqc_files.mix( QUALIMAP_BAMQC.out.results.collect{it[1]}.ifEmpty([]) )
```

Comment:
Like I said before, this needs to be down at the bottom just before MultiQC itself (you need to copy the if/else statement there too)